docs/apache-airflow/howto/run-behind-proxy.rst

```diff
@@ -62,7 +62,6 @@ Your reverse proxy (ex: nginx) should be configured as follow:
           server_name lab.mycompany.com;
 
           location /myorg/flower/ {
-              rewrite ^/myorg/flower/(.*)$ /$1 break;  # remove prefix from http header
```

Comment:
Update the comment from 58th line.

Reply:
Removed comment and merged nginx server blocks into one.